src/utils/axiosUtils.ts

```diff
@@ -0,0 +1,28 @@
+import axios, { AxiosRequestConfig, AxiosResponse } from 'axios';
+import HttpsProxyAgent from 'https-proxy-agent';
+
+/**
+ * This function is called to better handle axios.get calls with logic that allows
+ * the use of proxies. Not needed in tests unless specifically testing with proxies.
+ * Check https://github.com/axios/axios/issues/3459#issuecomment-766171276 for more info.
+ * @param url {string} - string representation of url to get
+ * @param responseType {any} - optional parameter to change the data type needed from get
+ * (ex. arraybuffer). In default it returns JSON
+ */
+export async function axiosGet(url: string, responseType?: any): Promise<AxiosResponse<any>> {
+  const httpsProxy = process.env.HTTPS_PROXY;
```

Comment:
This is just a question for my own understanding, but do you only need to look at the `HTTPS_PROXY` and not the `HTTP_PROXY` because we only make requests to `https` urls?

Reply:
Yep, we don't use the `HTTP_PROXY` with the `axios.get` functionality. Just to make sure, I tested only using `HTTP_PROXY` (keeping `HTTPS_PROXY` unset) and all the tests held up.